### ISSUE TITLE
Only showing 1 locales links

### ISSUE THREAD

**Issue by madebyshape** [fruitstudios/LinkIt#27]
I've got a LinkIt field currently in a Matrix, then inside a SuperTable. When I go to choose an entry it only shows the entries for 1 locale. Weirdly, this locale it shows entries for isn't the one we're currently in, editing the entry.

I tried creating just a standard LinkIt field to see if Matrix or SuperTable was causing the issue but creating a field on it's own doesn't work.

Is there meant to be locale options here:

<img width="1541" alt="screen shot 2016-04-05 at 13 47 50" src="https://cloud.githubusercontent.com/assets/10700908/14282167/155603a6-fb35-11e5-9fd8-c9b3fc0acabf.png">

Am I missing something? I did update from version 0.9.1 of Linkit if this is any help.

Thanks

**Comment by a-am** [comment]
I have found this as well.  If I have a multiple locale environment it only ever shows modal with entries/categories/assets associated with default locale not the locale selected in the entries edit screen.

**Comment by madebyshape** [comment]
Yeah, I've ended up having to revert back to just a standard Entries Field and a Plain Text Field until this is fixed :(

**Comment by Moorst** [comment]
I'm having this issue too.

I also upgraded from an older version of Linkit - is this solely a problem when upgrading or is everyone struggling with this?

Would be interested to know if this is being looked at or whether I'll need to use a different plugin (which will be a nightmare to swap out!)

**Comment by wateengoedidee** [comment]
We have the same issue, the entries are not shown in the selected language. This is different then Craft itself does. Why? And is there a solution coming?

**Comment by a-am** [comment]
@wateengoedidee if you follow the changes in this pull request https://github.com/fruitstudios/LinkIt/pull/38 you should get the functionality you need.

**Comment by wateengoedidee** [comment]
Hi,

Are you going to implement this in the next update? Otherwise we cannot
easily update LinkIt anymore.


Thanks in advance



Kind regards,

Tom Fischer

Bezoekadres Het Eeftink 11-21
7541 WH Enschede
T +316 24 89 41 19
E info@wateengoedidee.nl
W www.wateengoedidee.nl

Op 27 september 2016 bij 17:21:43, Adam Randlett (notifications@github.com)
schreef:

@wateengoedidee <https://github.com/wateengoedidee> if you follow the
changes in this pull request #38
<https://github.com/fruitstudios/LinkIt/pull/38> you should get the
functionality you need.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/fruitstudios/LinkIt/issues/27#issuecomment-249886852>,
or mute the thread

**Comment by mildlygeeky** [comment]
Would love to see #38 or something like it make it into core — love this plugin, but took a while to realize that it has issues when used inside subelements on localized sites. Thanks!

**Comment by samhibberd** [comment]
Folks, LinkIt is high on the list to get some love over Christmas. I will aim to resolve any issues and get a version ready to test with Craft 3.
